src/main/kotlin/me/lensvol/blackconnect/settings/BlackConnectProjectSettings.kt

```diff
@@ -18,6 +18,9 @@ class BlackConnectProjectSettings : PersistentStateComponent<BlackConnectProject
             ServiceManager.getService(project, BlackConnectProjectSettings::class.java)
     }
 
+    @Attribute
+    var https: Boolean = Constants.DEFAULT_USE_SSL
```

Comment:
Can `https` variable be renamed to something more descriptive like `useSSL` or `connectOverSSL`? As we would have to maintain that name for backward compatibility reasons. :)